src/containers/Ledgers/LedgerMetrics.tsx

```diff
@@ -0,0 +1,120 @@
+import { useTranslation } from 'react-i18next'
+import { Tooltip, useTooltip } from '../shared/components/Tooltip'
+import { renderXRP } from '../shared/utils'
+import PauseIcon from '../shared/images/ic_pause.svg'
+import ResumeIcon from '../shared/images/ic_play.svg'
+import './css/ledgerMetrics.scss'
+import { useIsOnline } from '../shared/SocketContext'
+import { useLanguage } from '../shared/hooks'
+
+const DEFAULTS = {
+  load_fee: '--',
+  txn_sec: '--',
+  txn_ledger: '--',
+  ledger_interval: '--',
+  avg_fee: '--',
+  quorum: '--',
+  nUnl: [],
+}
+
+export const LedgerMetrics = ({
+  data: suppliedData,
+  onPause,
+  paused,
+}: {
+  data: any
+  onPause: any
+  paused: boolean
+}) => {
+  const data = { ...DEFAULTS, ...suppliedData }
+  const { tooltip, showTooltip, hideTooltip } = useTooltip()
+  const { t } = useTranslation()
+  const isOnline = useIsOnline()
+  const language = useLanguage()
+
+  const renderPause = () => {
+    const Icon = paused ? ResumeIcon : PauseIcon
+    const text = paused ? 'resume' : 'pause'
+
+    return (
+      <div
+        tabIndex={0}
+        role="button"
+        className="pause-resume"
+        onClick={onPause}
+        onKeyUp={onPause}
+      >
+        <Icon className="icon" alt={t(text)} />
+        <span>{t(text)}</span>
+      </div>
+    )
+  }
+
+  if (data.load_fee === '--') {
+    data.load_fee = data.base_fee || '--'
+  }
+  delete data.base_fee
+  const items = Object.keys(data)
+    .map((key) => {
+      let content: any = null
+
+      let className = 'label'
+      if (data[key] === undefined && key !== 'nUnl') {
+        content = '--'
+      } else if (key.includes('fee') && !isNaN(data[key])) {
+        content = renderXRP(data[key], language)
+      } else if (key === 'ledger_interval' && data[key] !== '--') {
+        content = `${data[key]} ${t('seconds_short')}`
+      } else if (key === 'nUnl' && data[key]?.length === 0) {
+        return null
+      } else if (key === 'nUnl') {
+        content = data[key].length
+        className = 'label n-unl-metric'
+        return (
+          <div
+            role="link"
+            className="cell"
+            onFocus={() => {}}
+            onBlur={() => {}}
+            onMouseOver={(e) => showTooltip('nUnl', e, { nUnl: data.nUnl })}
+            onMouseOut={() => hideTooltip()}
+            tabIndex={0}
+            key={key}
+          >
+            <a
+              key={`link ${key}`}
+              href="https://xrpl.org/negative-unl.html"
```

Comment:
Should this link be "localized" to point to the proper language if the explorer is in Japanese for example?

Reply:
We don't link anywhere else based on the language.  I wouldn't know where to start since every site might have different rules like localstorage based or path based.